packages/flutter/lib/src/rendering/mouse_cursor.dart

```diff
@@ -236,6 +234,41 @@ abstract class MouseCursor with Diagnosticable {
   }
 }
 
+/// A special class that indicates that the region with this cursor defers the
+/// choice of cursor to the next region behind it.
+///
```

Comment:
Should explain that this is a singleton whose value is `[DeferredMouseCursor.instance]`.